src/SDK/Client/Client.ts

```diff
@@ -95,6 +95,7 @@ export class Client {
 
```

Comment:
Please rename `addresses` option to `dapiAddresses`

Reply:
Why?

Reply:
I see #110